decisioncentral/image.yaml

```diff
@@ -163,8 +164,8 @@ modules:
     - name: openshift-layer
     - name: openshift-passwd
     - name: jboss.container.util.logging.bash
-    - name: jboss.container.eap.prometheus.config
-      version: "7.2-kie"
+    - name: jboss.container.prometheus
```

Comment:
make sure this modules installs the needed jar and configures it properly like the previous one.

Reply:
Applied the previous one